uploading map to new relic
executing upload_map_file_to_new_relic
New Relic: map file upload attempt #1 (of 3)
Using URL: https://mobile-symbol-upload.newrelic.com
New Relic: map file upload attempt #2 (of 3)
Using URL: https://mobile-symbol-upload.newrelic.com
New Relic: map file upload attempt #3 (of 3)
Using URL: https://mobile-symbol-upload.newrelic.com
executing upload_dsym_to_new_relic
new relic: error "100" while uploading "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/SwiftyFORM.framework.dSYM-1672256576.zip" to "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/SwiftyFORM.framework.dSYM-1672256576new renew relic: error "401" wnew relic: error "401" while uploading "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/ContentRestrictionsKit.framework.dSYM-1672256575.zip" to "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/ContentRestrictionsKit.framework.dSYM-1672256575.zip"
new relic: error "401" while uploading "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/ContentRestrictionsKit.framework.dSYM-1672256575.zip" to "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/ContentRestrictionsKit.framework.dSYM-1672256575.zip"
oaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKidnew relic: error "401" while uploading "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/BLTNBoard.framework.dSYM-1672256573.zip" to "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaonew relic: error "100" while uploading "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/Kingfisher.framework.dSYM-1672256576.zip" to "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/Kingfisher.framework.dSYM-1672256576.zip"
mediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/FetchedResultsControllerCollectionViewUpdater.framework.dSYM-1672256577.zip"
onn)
  File "/Volumes/Developer/xcode_Apps/KKid/Pods/NewRelicAgent/NewRelic.xcframework/Resources/urllib3/connectionpool.py", line 1040, in _validate_conn
  new relic: error "401" while uploading "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/SwiftyJSON.framework.dSYM-1672256574.zip" to "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/SwiftyJSON.framework.dSYM-1672256574.zip"
rap_socket(sock, server_hostname=server_hostname)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Volumes/Developer/xcode_Apps/KKid/Pods/NewRelicAgent/NewRelic.xcframework/Resources/generateMap.py", line 271, in <module>
    r = requests.post("/".join((url, "map")), files=files, headers=headers)
  File "/Volumes/Developer/xcode_Apps/KKid/Pods/NewRelicAgent/NewRelic.xcframework/Resources/requests/api.py", line 117, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Volumes/Developer/xcode_Apps/KKid/Pods/NewRelicAgent/NewRelic.xcframework/Resources/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Volumes/Developer/xcode_Apps/KKid/Pods/NewRelicAgent/NewRelic.xcframework/Resources/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/Volumes/Developer/xcode_Apps/KKid/Pods/NewRelicAgent/NewRelic.xcframework/Resources/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/Volumes/Developer/xcode_Apps/KKid/Pods/NewRelicAgent/NewRelic.xcframework/Resources/requests/adapters.py", line 501, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
executing upload_dsym_to_new_relic
new relic: error "401" while uploading "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/Reachability.framework.dSYM-1672256576.zip" to "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/Reachability.framework.dSYM-new relic: errornew relic: error "401" while uploading "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/Reachability.framework.dSYM-1672256576.zip" to "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/Reachabilitynew relic: error "401" while uplnew relic: error "401" while uploading "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/Reachability.framework.dSYM-1672256576.zip" to "/Users/justinkumpe/Library/Developer/Xcode/DerivedData/KKid-ezimguypaoaiipctbjvrrhxxshlm/Build/Intermediates.noindex/ArchiveIntermediates/KKid/IntermediateBuildFilesPath/KKid.build/Release-iphoneos/KKid.build/Reachability.framework.dSYM-1672256576.zip"
